test/built-ins/String/prototype/codePointAt/return-abrupt-from-symbol-pos-to-integer.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 21.1.3.3
+description: >
+  Returns abrupt from ToInteger(pos)
+info: >
+  21.1.3.3 String.prototype.codePointAt ( pos )
+
+  1. Let O be RequireObjectCoercible(this value).
+  2. Let S be ToString(O).
+  3. ReturnIfAbrupt(S).
+  4. Let position be ToInteger(pos).
+  5. ReturnIfAbrupt(position).
+features: [Symbol]
+---*/
+
+assert.throws(TypeError, function() {
+  'abc'.codePointAt(Symbol(1));
```

Comment:
In order to avoid false positives and to improve readability, it's generally best to limit the code within the callback to `assert.throws` as much as possible. In this case, I'm thinking that the test could pass if the implementation incorrectly threw a TypeError from `Symbol(1)` or `Symbol()` in the case of `return-abrupt-from-this-as-symbol.js` (maybe because the description is not a string? I dunno) It's definitely unlikely (and not something we can protect against in all cases), but it can't hurt to move the constructor invocation outside as `var s = new Symbol(1);`. It also makes the test slightly more readable, because there's less ambiguity as to the expression that is expected to generate the error.